zero/core/utils.py

```diff
@@ -343,10 +343,9 @@ def get_service_from_metadata_extension_type(extension_type: WorkspaceExtension)
     if extension_type in {
         WorkspaceExtension.TEXT_GENERATION,
         WorkspaceExtension.VISION,
+        WorkspaceExtension.ADAPT,
```

Comment:
we updated the conditionals; lets also update the names of the keys, variables and imports accordingly

Reply:
what? 🤔

Reply:
the name of variables and imports; change them accordingly

Reply:
done.